### PR TITLE
Add unit test on pkg/chaosdaemon

### PR DESCRIPTION
**Pull request by lucklove**
### What problem does this PR solve? 
Add unit test

### What is changed and how does it work?
Add unit test
### Check List 

Tests 

 - Unit test

Code changes

 - Has Go code change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

**Comment by zhouqiang-cl** [comment]
@lucklove please fix CI

File: pkg/chaosdaemon/util.go [hunk 1]
```diff
@@ -103,27 +104,50 @@ func (c ContainerdClient) GetPidFromContainerID(ctx context.Context, containerID
 	return task.Pid(), nil
 }
 
-// CreateContainerRuntimeInfoClient creates a container runtime information client.
-func CreateContainerRuntimeInfoClient(containerRuntime string) (ContainerRuntimeInfoClient, error) {
-	// TODO: support more container runtime
+// newDockerClient wrap dockerclient.NewClient with mock point
```
Comment (cwen0) [inline pkg/chaosdaemon/util.go]: // newDockerclient returns a dockerlient.NewClient with mock points

File: pkg/chaosdaemon/util.go [hunk 2]
```diff
@@ -103,27 +104,50 @@ func (c ContainerdClient) GetPidFromContainerID(ctx context.Context, containerID
...
+	}
+	if client := mock.On("MockDockerClient"); client != nil {
+		return client.(DockerClientInterface), nil
+	}
+
+	// The real logic
+	return dockerclient.NewClient(host, version, client, httpHeaders)
+}
 
+// newDockerClient wrap dockerclient.NewClient with mock point
```
Comment (cwen0) [inline pkg/chaosdaemon/util.go]: ditto

**Pull request merged**